schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,522 @@
+
+[cellranger 2020-A (July 7, 2020) release]: https://support.10xgenomics.com/single-cell-gene-expression/software/release-notes/build
+
+## `obs` (Cell Metadata)
+
+`obs` is a [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html).
+
+Curators MUST annotate the following columns in the `obs` dataframe:
+
+### assay_ontology_term_id
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>assay_ontology_term_id</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator<td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>categorical with <code>str</code> categories. An assay based on 10X Genomics products SHOULD either be <a href="http://www.ebi.ac.uk/efo/EFO_0008995"><code>"EFO:0008995"</code></a> for <i>10x technology</i> or <b>preferably</b> its most accurate child. Other assays SHOULD be the most accurate child of either <a href="http://www.ebi.ac.uk/efo/EFO_0002772">EFO:0002772</a> for <i>assay by molecule</i> or <a href="http://www.ebi.ac.uk/efo/EFO_0010183">EFO:0010183</a>  for <i>single cell library construction</i>.<br><br>
+        If there is not an exact match for the assay, clarifying text MAY be appended in parentheses to the most accurate term. For example, the sci-plex assay could be curated as <code>"EFO:0010183 (sci-plex)"</code>.<br><br>Recommended keys for specific assays:
+          <br><br>
+          <table>
+          <thead>
+          <tr>
+          <th>For</th>
+          <th>Use</th>
+          </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><i>10x 3' v2</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0009899">EFO:0009899</a></td>
+            </tr>
+            <tr>
+              <td><i>10x 3' v3</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0009922">EFO:0009922</a></td>
+            </tr>
+            <tr>
+              <td><i>10x 5' v1</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0011025">EFO:0011025</a></td>
+            </tr>
+            <tr>
+              <td><i>Smart-seq</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0008930">EFO:0008930</a></td>
+            </tr>
+            <tr>
+              <td><i>Smart-seq2</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0008931">EFO:0008931</a></td>
+            </tr>
+          </tbody></table>
+        </td>
+    </tr>
+</tbody></table>
+<br><br>
+
+### cell_type_ontology_term_id
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>cell_type_ontology_term_id</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator<td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>categorical with <code>str</code> categories. This MUST be a CL term.
+        </td>
+    </tr>
+</tbody></table>
+<br><br>
+
+### development_stage_ontology_term_id
```

Comment:
I know the general life stage of my donor, but not the specific age at the resolution that is being asked of me here. How do I submit the following...
- My samples are derived from embryonic stem cells. I know 'embryonic', but not more specific than that
- I reused publication data that specified the samples were 'adult', but nothing more specific than that
- My IRB prevents me from sharing exact ages so I have the following values '40-50', '50-60', '60-70', '90+'

Reply:
These are valid edge cases. What would cover all of them is to follow similar requirements as `assay_ontology_term_id`. I bolded suggested changes:

-------

<th>Value</th>
        <td>categorical with <code>str</code> categories. If unavailable, this MUST be <code>"unknown"</code>.<br><br>If <code>organism_ontolology_term_id</code> is <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i>, this MUST be <b>the most accurate HsapDv term with the following STRONGLY RECOMMENDED specifications</b>:
          <br><br>
          <table>
          <thead>
          <tr>
          <th>For</th>
          <th>Use</th>
          </tr>
          </thead>
          <tbody>
            <tr>
              <td>Embryonic stage</td>
              <td>A term from the set of <a href="http://www.ontobee.org/search?ontology=HSAPDV&keywords=carnegie&submit=Search+terms">Carnegie stages 1-23</a> (up to 8 weeks after conception; e.g. <a href="http://purl.obolibrary.org/obo/HsapDv_0000003">HsapDv:0000003</a>)</td>
            </tr>
            <tr>
              <td>Fetal development</td>
              <td>A term from the set of <a href="http://www.ontobee.org/search?ontology=HSAPDV&keywords=post-fertilization&submit=Search+terms">9 to 38 week post-fertilization human stages</a> (9 weeks after conception and before birth; e.g. <a href="http://purl.obolibrary.org/obo/HsapDv_0000046">HsapDv:0000046</a>)</td>
            </tr>
            <tr>
              <td>After birth for the first 12 months</td>
              <td>A term from the set of <a href="http://www.ontobee.org/search?ontology=HSAPDV&keywords=month-old&submit=Search+terms">1 to 12 month-old human stages</a> (e.g. <a href="http://purl.obolibrary.org/obo/HsapDv_0000174">HsapDv:0000174)</a></td>
            </tr>
            <tr>
              <td>After the first 12 months post-birth</td>
              <td>A term from the set of <a href="http://www.ontobee.org/search?ontology=HSAPDV&keywords=year-old&submit=Search+terms">year-old human stages</a> (e.g. <a href="http://purl.obolibrary.org/obo/HsapDv_0000246">HsapDv:0000246)</a></td>
            </tr>
          </tbody></table>
          <br><br>If <code>organism_ontolology_term_id</code> is <code>"NCBITaxon:10090"</code> for <i>Mus musculus</i></code>, this MUST be <b>the most accurate MmusDv term with the following STRONGLY RECOMMENDED specifications:</b>
          <br><br>
          <table>
          <thead>
          <tr>
          <th>For</th>
          <th>Use</th>
          </tr>
          </thead>
          <tbody>
            <tr>
              <td>From the time of conception to 1 month after birth</td>
              <td>A term from the set of <a href="http://www.ontobee.org/search?ontology=MMUSDV&keywords=theiler+stage&submit=Search+terms">Theiler stages</a> (e.g. <a href="http://purl.obolibrary.org/obo/MmusDv_0000003">MmusDv:0000003</a>)</td>
            </tr>
            <tr>
              <td>From 2 months after birth</td>
              <td>A term from the set of <a href="http://www.ontobee.org/search?ontology=MMUSDV&keywords=month-old&submit=Search+terms"> month-old stages</a> (e.g. <a href="http://purl.obolibrary.org/obo/MmusDv_0000062">MmusDv:0000062)</a></td>
            </tr>
          </tbody></table>

Reply:
Perfect

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/d3b0364c33a02f10ac088da0805bab39e0d1cb5f